LBHackney-IT/lbh-social-care
# update create person form

Update create new person

To test:

- Search for a person
- Click create new person
- Complete form
- Search for new created person
- Also check save functionality is working

Review note: Tuomo mentioned that ageGroup should be contextFlag
Additional styling has been added to get the select aligned with the inputs. 

![createNewPerson](https://user-images.githubusercontent.com/9123541/106484610-b7bf0f80-64a7-11eb-928c-dde7952978b1.png)
![Screenshot 2021-02-01 at 15 53 23](https://user-images.githubusercontent.com/9123541/106484769-d6bda180-64a7-11eb-8c07-a9e272ac4c38.png)
